### DISCUSSION TITLE
Only one faq schema for several faq blocks

### DISCUSSION THREAD

**Discussion by splinksl** [elementor/elementor#29594] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When we add several FAQ blocks on the same page and activate the generation of schema markup, several independent markups are generated and Google detects it as an error.

### Describe the solution you'd like.

I would like only a schema markup to be generated grouping all the existing FAQs on the page

### Describe alternatives you've considered.

Support tells me that this could be solved with an extra plugin, but I don't want to add more plugins to the website because it already has too many and it will slow it down even more.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by rami-elementor** [comment]
Please see https://developers.google.com/search/blog/2023/08/howto-faq-changes
